core/domain/question_services.py

```diff
@@ -153,6 +153,63 @@ def get_question_by_id(question_id, strict=True):
         return None
 
 
+def get_question_skill_links_of_skill(skill_id):
+    """Returns a list of QuestionSkillLink domains of a particular skill ID.
+
+    Args:
+        skill_id: str. ID of the skill
+
+    Returns:
+        list(QuestionSkillLink)|None. The list of question skill link
+        domains that are linked to the skill ID. None if it doesnt exist.
+    """
+
+    question_skill_link_model_list = (
+        question_models.QuestionSkillLinkModel.get_question_skill_models_of_skill( #pylint: disable=line-too-long
+            skill_id)
+    )
+    question_skill_link_domains = [
```

Comment:
question_skill_links

(see elsewhere; terminology for referring to domain objects is to just refer to the object without appending _domain_object or _domain).